### DISCUSSION TITLE
filter automations by state

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#8889] [Automations / scripts / scenes]
on /config/automation/dashboard it would be very helpful to be able to display the automations by state (on/off).
right now they are alphabetically listed, which is fine, but on and off automations are listed mixed because of that
selecting last_triggered messes the alphabetical listing.

not sure if this is the correct category, could have selected Lovelace or Other...

**Comment by jpbaril** [comment]
Sorting and filtering have recently been improved in Home Assistant but filtering automations by state is still missing.

That would be useful as I often create multiple variations of a same automation to see which one works best (or try multiple Blueprints for that same reason), and so I toggle them on and off without deleting them. But it can thus get cluttered. It would want to be able to hide those that are disabled.

**Reply by Mariusthvdb** [reply]
marking as solved /closing as this was implemented some while ago

**Comment by ildar170975** [comment]
https://github.com/home-assistant/frontend/pull/23977 added a possibility to sort by a state, not to filter.
Although a possibility to sort allowed users to separate enabled/disabled automations.

**Reply by Mariusthvdb** [reply]
yes, you're right. let me mark as unanswered again.....
